<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget {
    background-image: url(:/imagenes/Imagenes/Fondo.png);
    background-repeat: no-repeat;
    background-position: center;
    background-size: cover;
}</string>
   </property>
   <widget class="QLabel" name="labelCantPuntos">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>150</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Puntos</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMultasTt">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>260</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Multas</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNivel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>NIVEL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDenegar">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>130</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color: rgba(255, 255, 255, 0); border: none; }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAceptar">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color: rgba(255, 255, 255, 0); border: none; }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDatos">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>400</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Revisar Documentos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>420</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color: rgba(255, 255, 255, 0); border: none; }
</string>
    </property>
    <property name="text">
     <string> </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>757</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
